registrations/tasks.py

```diff
@@ -220,25 +296,60 @@ def create_subscriptionrequests(self, registration):
         """ Create SubscriptionRequest(s) based on the
         validated registration.
         """
+
+        mother_short_name = get_messageset_short_name(
+            registration.stage, 'mother', registration.data["msg_type"],
+            registration.data["preg_week"]
+        )
+
+        if 'voice_days' in registration.data:
+            voice_days = registration.data["voice_days"]
+            voice_times = registration.data["voice_times"]
+        else:
+            voice_days = None
+            voice_times = None
+
+        if 'preg_week' in registration.data:
+            weeks = registration.data["preg_week"]
+        else:
+            weeks = registration.data["baby_age"]
+
+        mother_msgset_id, mother_msgset_schedule, next_sequence_number =\
+            get_messageset_schedule_sequence(
+                mother_short_name, weeks, voice_days, voice_times)
         mother_sub = {
             "contact": registration.mother_id,
-            "messageset_id": 1,  # TODO
-            "next_sequence_number": 1,  # TODO
+            "messageset_id": mother_msgset_id,
+            "next_sequence_number": next_sequence_number,
             "lang": LANG_CODES[registration.data["language"]],
```

Comment:
we need to not need this mapping because we're using proper lang codes. Opened https://github.com/praekelt/mama-ng-jsbox/issues/105 for the other side of this, go to using the codes as-is in the submission.